src/main/java/uk/gov/hmcts/cmc/claimstore/models/otherparty/TheirDetails.java

```diff
@@ -43,7 +45,7 @@
     @NotNull
     private final Address address;
 
-    @Email
+    @Email(regexp = "\\S+")
```

Comment:
According to the spec `regex` property is used to specify an additional regular expression the annotated string must match. The default is any string ('.*').